Sources/StreamVideoSwiftUI/CallView/VideoParticipantsView.swift

```diff
@@ -325,7 +326,10 @@ public struct VideoCallParticipantView<Factory: ViewFactory>: View {
     var customData: [String: RawJSON]
     var call: Call?
 
-    @State private var isUsingFrontCameraForLocalUser: Bool = false
+    private var isLocalParticipant: Bool
+
+    private var callSettingsPublisher: AnyPublisher<CallSettings, Never>?
```

Comment:
I think we discussed this once - is it safe that this is recreated on every init?

Reply:
I don't see any reason why that should cause issues? Publishers are just reference boxes. They are cheap to create and they only start pushing values when they receive their first subscription. We are safe here.